lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # Car behaviour
+  class Car
+    attr_reader :color, :model, :year, :current_speed
+
+    def initialize(car)
```

Comment:
car it's sounds like you need pass a car object, but in fact you need to pass only cars attributes like a hash. So the better way will be named it  - `car_options` or `car_attributes` and of set default as a `{}`
`def initialize(car_attributes = {})`